examples/pubsub/sub/sub.go

```diff
@@ -46,12 +46,16 @@ var importantSubscription = &common.Subscription{
 func main() {
 	s := daprd.NewService(":8080")
 
-	if err := s.AddTopicEventHandler(defaultSubscription, eventHandler); err != nil {
-		log.Fatalf("error adding topic subscription: %v", err)
-	}
+	bulkSubscribe := true
 
-	if err := s.AddTopicEventHandler(importantSubscription, importantEventHandler); err != nil {
-		log.Fatalf("error adding topic subscription: %v", err)
+	if bulkSubscribe {
+		if err := s.AddBulkTopicEventHandler(defaultSubscription, eventHandler, 10, 100); err != nil {
+			log.Fatalf("error adding topic subscription: %v", err)
+		}
+	} else {
+		if err := s.AddTopicEventHandler(defaultSubscription, eventHandler); err != nil {
+			log.Fatalf("error adding topic subscription: %v", err)
+		}
```

Comment:
I guess my concern is that we're hijacking the non-bulk topic event example, rather than adding an if-statement that will effectively always use the `AddBulkTopicEventHandler`. We could look at maybe implementing two topics, one for the original subscription method and one for bulk.